Source/ACE/Command/Handlers/DebugCommands.cs

```diff
@@ -118,7 +118,13 @@ public static void spacejump(Session session, params string[] parameters)
         [CommandHandler("createlifestone", AccessLevel.Developer, CommandHandlerFlag.RequiresWorld)]
         public static void CreateLifeStone(Session session, params string[] parameters)
         {
-            LandblockManager.AddObject(AdminObjectFactory.CreateLifestone(session.Player.Position.InFrontOf(3.0f)));
+            LandblockManager.AddObject(LifestoneObjectFactory.CreateLifestone(509, session.Player.Position.InFrontOf(3.0f), LifestoneType.Original));
+        }
+
+        [CommandHandler("createportal", AccessLevel.Developer, CommandHandlerFlag.RequiresWorld)]
+        public static void CreatePortal(Session session, params string[] parameters)
+        {
+            LandblockManager.AddObject(PortalObjectFactory.CreatePortal(1234, session.Player.Position.InFrontOf(3.0f),"Test Portal",PortalType.Purple));

```

Comment:
whitespace consistency - spaces after commas.  not going to block a PR over it, but want you to know.  stylecop rules coming down the pipe will enforce this in the future.